Misc/NEWS.d/next/Library/2024-07-13-13-20-43.gh-issue-107001.fRSPOX.rst

```diff
@@ -0,0 +1 @@
+Add :func:`copy_kwargs` to *typing* that copies/applies the ParameterSpec from one function to another.
```

Comment:
This causes a [reference warning](https://github.com/python/cpython/actions/runs/9920273837/job/27406982438?pr=121693) because Sphinx doesn't know about `copy_kwargs` here.

Try prefixing with `~typing.` so it knows the full reference -- the `~` tells Sphinx to only display `copy_kwargs` but still link to it:

```suggestion
Add :func:`~typing.copy_kwargs` to *typing* that copies/applies the ParameterSpec from one function to another.
```

---

If you like, you can add a "Patch by [your name]." at the end of the NEWS entry.

---

Please also document this addition in the "What's New" under typing:

https://docs.python.org/3.14/whatsnew/3.14.html#improved-modules